packages/react-components/package.json

```diff
@@ -10,36 +10,55 @@
     "build:watch": "tsc --build --watch",
     "build:storybook": "tsc --build stories && ../../node_modules/.bin/build-storybook -s stories",
     "clean": "tsc --build --clean",
-    "test": "jest --watch",
-    "test:coverage": "jest --ci --coverage",
+    "test": "./node_modules/karma/bin/karma start karma.conf.js --singleRun",
+    "test:watch": "./node_modules/karma/bin/karma start karma.conf.js",
```

Comment:
I think we can use `karma` directly, scripts are ran with the bin PATH added.

Reply:
I'll try it, I had to install it globally. To make it work on my pc.

Reply:
It worked!